packages/@expo/cli/package.json

```diff
@@ -77,7 +77,7 @@
     "fs-extra": "~8.1.0",
     "getenv": "^1.0.0",
     "glob": "^10.4.2",
-    "internal-ip": "^4.3.0",
+    "internal-ip": "6.1.0",
```

Comment:
I think ^6.2.0 works, looking at that version’s package.json.

Reply:
It contains `packge.json:exports` without `package.json:main`, which feels like it'd cause trouble in some older Node versions

Reply:
Gotcha. Seems fine then.